homeassistant/components/rki_covid/__init__.py

```diff
@@ -0,0 +1,59 @@
+"""RKI Covid numbers integration."""
+import logging
+
+from homeassistant.components.rki_covid.coordinator import RkiCovidDataUpdateCoordinator
+
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.const import CONF_SCAN_INTERVAL, Platform
+
+from .const import DOMAIN, DATA_CONFIG_ENTRY, DEFAULT_SCAN_INTERVAL
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up the RKI covid numbers integration from a config entry."""
+    _LOGGER.debug("async_setup_entry from a config entry")
+
+    # create coordinator instance
+    coordinator = RkiCovidDataUpdateCoordinator(hass)
+
+    # trigger initial refresh
+    await coordinator.async_config_entry_first_refresh()
+
+    # put data into
+    hass.data.setdefault(DOMAIN, {})
+    hass.data[DOMAIN].setdefault(DATA_CONFIG_ENTRY, {})
+    hass.data[DOMAIN][DATA_CONFIG_ENTRY][entry.entry_id] = coordinator
```

Comment:
We don't need an additional dictionary here, as it seems there is no additional data put in.

```suggestion
    # put data into
    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = coordinator
```